ruby_project/classes/nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
```

Comment:
Kindly add a more descriptive message when an error raises. Consider doing something like this:
```
raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
```
Or you can customize it to your preference.